ch02-git-basics-chapter.asc

```diff
@@ -20,7 +19,7 @@ include::book/02-git-basics/sections/tagging.asc[]
 
 include::book/02-git-basics/sections/aliases.asc[]
 
-=== Summary
+=== Özet
 
-At this point, you can do all the basic local Git operations – creating or cloning a repository, making changes, staging and committing those changes, and viewing the history of all the changes the repository has been through.
-Next, we'll cover Git's killer feature: its branching model.
+Bu noktada, tüm temel yerel Git işlemlerini yapabilirsiniz - bir depoyu oluşturmak veya kopyalamak, değişiklik yapmak, bu değişiklikleri sahnelemek ve yerine getirmek ve deponun geçtiği tüm değişikliklerin geçmişini görüntülemek.
+Sonra, Git'in muhteşem özelliğini ele alacağız: yayılma modeli.
```

Comment:
Branching daha ziyade "dallanma modeli" olabilir. Zira bir branch diğerinden dallanarak oluşturulduğu için çok daha uygun bir karşılık.